projects/packages/backup/src/js/hooks/useConnection.js

```diff
@@ -23,6 +23,11 @@ export default function useConnection() {
 	const [ price, setPrice ] = useState( 0 );
 	const [ priceAfter, setPriceAfter ] = useState( 0 );
 
+	const checkSiteHasBackupProduct = useCallback(
+		() => apiFetch( { path: '/jetpack/v4/has-backup-plan' } ).then( res => res ),
```

Comment:
is the `.then( res => res )` necessary here?

Reply:
Cleaned this up, it wasn't.